clicommand/agent_start.go

```diff
@@ -84,6 +84,7 @@ type AgentStartConfig struct {
 	HealthCheckAddr            string   `cli:"health-check-addr"`
 	MetricsDatadog             bool     `cli:"metrics-datadog"`
 	MetricsDatadogHost         string   `cli:"metrics-datadog-host"`
+	TracingDatadogAddr         string   `cli:"tracing-datadog-addr"`
```

Comment:
As a separate comment, I think this should be split up into a `bool` and a `string`, where having it enabled but with the address set to the empty string allows us to fall back on DataDog's default environment variables like DD_AGENT_HOST etc. This means that things will "just" work in environments that are already configured to use those, such as things running in a K8s environment

Reply:
I turned this into `tracing-backend`. By default an empty string will mean no tracing. Setting it to 'datadog' will enable it. This lets people in the future add support for other tracing libs, like jaeger. And specific tracing lib configuration will be done through the common env vars like you mentioned. Verified this works on my local setup.

Reply:
Thanks very much!